src/atomate2/abinit/jobs/base.py

```
@@ -224,7 +278,7 @@ def get_response(
             return Response(
                 output=task_document,
                 stop_children=True,
-                stop_jobflow=False,
+                stop_jobflow=self.stop_jobflow_on_failure,
```

I don't remember if we already discussed this and I see that it was already like this in the original implementation. But why, in case of unconverged, is this stopping everything instead of raising? 

In any case, the variable name is somewhat misleading, because it is not upon failure that the stop is issued, but only if it not converged with a certain number of restarts. If there is an error in the job execution it will not arrive here, isn't it?

---

I don't understand what you mean by "raise"... 

Except for the convergence errors, the other ABINIT errors are caught in the taskdoc but not treated as far as I know because only the convergence errors are stored in ```self.critical_events``` and used to filter the ```task_doc.event_report```. I guess the ```stop_jobflow_on_failure``` was termed this way because it could also be used when handling other ABINIT errors. Also, I would say that the inability to converge after a given number of times can be considered as a failure.

---

This is still to be checked, do you still want to do it?

these are the notes of what we discussed:

"""
do smth like "should_stop_children" of VASP workflows to handle if stop_children or not based on the output.state, let's try the same with a new "should_stop_jobflow" which by default should raise an error for any error event (NOT WARNINGS) that is not a convergence error, BUT let's wait for Guido to do some tests and figure out what happens
"""